### DISCUSSION TITLE
Idea: Custom statuses like slack does

### DISCUSSION THREAD

**Discussion by Cybernetks** [jschuur/coworking.party#89] [Ideas]
So you could choose from the list but also add a custom one if you want.

**Comment by jschuur** [comment]
A good idea, @SchabrechtsK! I could see some kind of unique style to recognise custom statuses like a border around the badge or even a (random) gradient background color.

Custom statuses could also be marked as 'private', so they're tracked for the user but not shared on a per status basis.
